docs/reducing_variance.md

```diff
@@ -39,6 +39,27 @@ The benchmarks you subsequently run will have less variance.
 
 <a name="reducing-variance" />
 
+## Disabling ASLR
+
+If you see this error:
+
+```
+***WARNING*** ASLR is enabled, the results may have unreproducible noise in them.
+```
+
+you might want to disable the ASLR security hardening feature while running the
+benchmark.
+
+To globally disable ASLR, run
```

Comment:
these instructions only apply to linux i think?  windows also has ASLR available.

Reply:
(i assume you don't care given the change in the code, but it might be worth mentioning in the documentation)